lib/tsParser.js

```diff
@@ -19,15 +19,17 @@ function TSParser(filePaths, config) {
   this.constructs = [];
   config = config || {};
   var options = {
-    mode: 'Modules',
+    mode: 'modules',
     logger: 'console',
-    target: config.tstarget || 'ES6',
-    module: 'CommonJS',
+    target: config.target || 'ES6',
+    module: 'commonjs',
```

Comment:
any reason changing it from camel to all lower case? 
options from ts document https://www.typescriptlang.org/docs/handbook/compiler-options.html is camel

Reply:
See http://json.schemastore.org/tsconfig. These names are case-insensitive. 

Typedoc README also references them as lower-cases: https://github.com/TypeStrong/typedoc